core/js/src/main/scala/bobcats/HotpPlatform.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bobcats
+
+private[bobcats] trait HotpPlatform[F[_]] {}
+
+private[bobcats] trait HotpCompanionPlatform {}
```

Comment:
I have added these just in case - is it better to do that for bincompat or should they be omitted?

Reply:
These are private anyway so there aren't bincompat issues. Also I'm not in any rush to do a stable release 😬